tests/conditional_formatting/conditional_formatting_plugin.test.ts

```diff
@@ -675,7 +675,7 @@ describe("conditional format", () => {
     });
   });
 
-  test("cannot send invalid arguments to conditional format rules command", () => {
+  test("cannot send invalid arguments to *move* conditional format rules command", () => {
```

Comment:
\*move\* ?

Reply:
erf, priority was the word I was looking for